esphome/components/online_image/online_image.h

```diff
@@ -119,6 +126,12 @@ class OnlineImage : public PollingComponent,
 
   uint8_t *buffer_;
   DownloadBuffer download_buffer_;
+  /**
+   * This is the *initial* size of the download buffer, not the current size.
+   * The download buffer can be resized at runtime; the download_buffer_size_ will
```

Comment:
```suggestion
   * The download buffer can be resized at runtime; the download_buffer_initial_size_ will
```

Maybe the constructor parameter should also be renamed?

Reply:
Thanks for finding this one.

W.r.t. renaming the constructor parameter; I'd rather not. The constructor parameter is actually used to set the size of the download buffer (as the name says); just in case of Jpeg decoder, the image needs to be fully downloaded before decoding (because of the way the decoder works), so the initial size value is still needed later on.